browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1824,7 +1824,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		if (data.enabled === false || data.enabled === 'false')
 			$(listbox).attr('disabled', 'disabled');
 
-		$(listbox).change(function() {
+		$(listbox).change(() => {
 			if ($(this).val())
 				builder.callback('combobox', 'selected', data, $(this).val()+ ';' + $(this).children('option:selected').text(), builder);
 		});
```

Comment:
what is the `this` referring to here? I worry that by changing this to an arrow function you may have altered its meaning

Reply:
ah, good catch. It is a jQuery callback where `this` was a `listbox` / target of the callback.
I think they do `.bind(listbox)` on the passed function object so it should work.
I checked it in Format -> Character -> Style. Seems to work.

Reply:
good catch, fixed

Reply:
can you bind arrow functions? I was under the impression that this isn't a thing you can do and from a quick test node seems to agree on this...
![image](https://github.com/CollaboraOnline/online/assets/34243578/19be5593-8b69-4692-9dc2-74853a473aeb)

Reply:
in contrast, here's what happens if you do the same thing without using an arrow function
![image](https://github.com/CollaboraOnline/online/assets/34243578/fdf73ca6-dacc-4012-8fc8-347d826d91e6)